Makefile

```diff
@@ -44,8 +44,31 @@ docker-buildx:
 
 # Needs `DOPPLER_TOKEN` env var to fetch secrets from Doppler's API
 # Learn more at https://docs.doppler.com/docs/enclave-service-tokens
+# usage: make docker-run-prod DOPPLER_TOKEN=dp.st.XXXX
 docker-run:
-	docker container run -it --rm -e DOPPLER_TOKEN=$(DOPPLER_TOKEN) -p 3000:3000 dopplerhq/yodaspeak:latest
+	docker container run \
+		-it \
+		--init \
```

Comment:
Do we need `--init`?

Reply:
Not strictly, but it's [a good best practice](https://github.com/krallin/tini/issues/8#issuecomment-146135930) so would prefer to have it included.

Reply:
This lead me down a bit of a rabbit hole earlier, which ultimately lead to this PR: https://github.com/DopplerHQ/cli/pull/139. I'd prefer if we left out `--init` here since I'm going to update our docs to note that our images don't need it (but custom images do). It's definitely a nit though.

Reply:
Ok got it! Will remove it.